fix: typos in README (#8)
Backbase/.github: profile/README.md

# Backbase Open Source

> Welcome to Backbase's open source organisation. Regardless of if you are a customer, partner, or employee you can create and share with everyone in the Backbase Community via this organisation.

Reviewer: Maybe `Regardless of if ...` -> `Regardless if` or `Regardless of whether`? (Because is sounds little weird otherwise.)

Reviewer: The heading is `Backbase Open Source`, not `Backbase's Open Source`; hence, would be `Backbase Open Source organisation` instead.

Author: I agree, done ✅

Reviewer 2: Would you mind while updating to also change the links?

https://backbase.io/developers/

and

https://backbase.io/learning/

Also, I believe there are 2 versions of this text, one for the public view and one for the members. So this should be copied their too.

Thank you

Author: @craigwalkeruk links are updated ✅

[I searched for the title of the README on GitHub to find a second page/text](https://github.com/search?q=Welcome+to+Backbase%27s+open+source+organisation.+Regardless+to+if+you+are+a+customer%2C&type=code) but this is the only file I found so far. 
Maybe you could share a link to the 2nd file that should be updated as well?

Author: Hi @ugur-bb @craigwalkeruk, 
I adressed all comments, except one about the second version of this text that needs to be updated. Unfortunately I couldn't find it myself. Could you please help with it?

> You'll find repo's here directly maintained by our Backbase teams and essential tools used as part of our developer experience. See [backbase.io/learning](https://backbase.io/learning/) to pick up a course on our best practices and [backbase.io/developers](https://backbase.io/developers/) to see the official developer documentation.

Reviewer 3: ```suggestion
Some of the repositories you will find here are directly maintained by our Backbase teams, as they contain essential tools used as part of our developer experience. See [backbase.io/learning](https://backbase.io/learning/) to pick up a course on our best practices and [backbase.io/developers](https://backbase.io/developers/) to see the official developer documentation.
```

> You'll also find community created connections to 3rd parties and accelerators shared by projects that can be used to speed up projects, these are not supported by Backbase's official SLA but by the community. For more info see our [github organisation contribution guidelines](https://github.com/Backbase/.github/blob/main/CONTRIBUTING.md)

Reviewer 3: ```suggestion
You'll also find community created 3rd party connectors and accelerators that can be used to speed up projects. All the code found in this space is not supported according to the official Backbase SLA, but by the community. For more info see our [github organisation contribution guidelines](https://github.com/Backbase/.github/blob/main/CONTRIBUTING.md)
```
